frontend/src/component/project/Project/ProjectStatus/ProjectStatusModal.tsx

```diff
@@ -16,15 +16,15 @@ const ModalContentContainer = styled('section')(({ theme }) => ({
     backgroundColor: theme.palette.background.default,
     display: 'flex',
     flexFlow: 'column',
-    gap: theme.spacing(4),
+    gap: theme.spacing(2),
     paddingInline: theme.spacing(4),
-    paddingBlock: theme.spacing(10),
+    paddingBlock: theme.spacing(4),
```

Comment:
Think this is 3.75 (30px) to align it correctly. Might be worth double checking.

This also reduces the padding at the bottom, but that's probably intentional.
```suggestion
    paddingBlock: theme.spacing(3.75),
```